crates/aranya-client-capi/src/imp/mod.rs

```diff
@@ -1,8 +1,11 @@
+pub mod afc;
 pub mod aqc;
```

Comment:
Should these go behind feature flags?
```suggestion
#[cfg(feature = "afc")]
pub mod afc;
#[cfg(feature = "aqc")]
pub mod aqc;
```

Reply:
They are. It's set using inner attributes